cc.retroshare.retroshare-gui.yaml

```diff
@@ -9,6 +9,9 @@ finish-args:
   - --socket=wayland
   - --socket=x11
   - --share=network
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=org.freedesktop.Notifications
```

Comment:
Just to clarify since your comment only mentions the tray. This application does have Unity integration and Notifications? Otherwise those aren't needed.

Reply:
I'm sure that org.kde.StatusNotifierWatcher is needed for appindicator to work on KDE. A quick test removing it here broke the systray support on retroshare, and a quick search on KDE docs confirmed that indeed this namespace also deals with appindicators.

What I was not entirely sure is if org.freedesktop.Notifications and com.canonical.Unity does the same for GNOME based desktops and Unity. And looks that it does not. Retroshare do have notification, but uses a custom implementation. After removing both here, notifications kept working all the same.

Sniffing dbus with dbus-monitor did show that it uses only org.kde.StatusNotifierWatcher and (quite unexpectedly) com.canonical.dbusmenu.

So, removing com.canonical.Unity and org.freedesktop.Notifications.